dockerfiles/logstash/logstash.conf-template

```diff
@@ -17,14 +17,14 @@ filter {
   if [@metadata][DEBUG] != 'true' {
     ruby {
       init => "@ordernum = 0"
-      code => "@ordernum += 1; event['scale_order_num'] = @ordernum; event['scale_task'] = event['program'].sub(%r{^docker/}, ''); event['scale_job_exe'] = event['scale_task'].sub(%r{_[^_]*$}, ''); event['scale_node'] = event['logsource']; event['stream'] = event['severity'] == 3 ? 'stderr' : 'stdout'"
-      remove_field => ["host", "priority", "timestamp8601", "logsource", "program", "pid", "severity", "facility", "timestamp", "facility_label", "severity_label"]
+      code => "@ordernum += 1; tag_items = event['program'].split('++'); event['scale_order_num'] = @ordernum; event['scale_task'] = tag_items[0].sub(%r{^docker/}, ''); event['scale_job_exe'] = event['scale_task'].sub(%r{_[^_]*$}, ''); event['scale_node'] = event['logsource']; event['stream'] = event['severity'] == 3 ? 'stderr' : 'stdout'; event['job_type'] = tag_items[1]"
+     remove_field => ["host", "priority", "timestamp8601", "logsource", "program", "pid", "severity", "facility", "timestamp", "facility_label", "severity_label", "job_type"]
```

Comment:
Looks like an indentation inconsistency here.

Reply:
With this change we are going to need to be a bit more careful about maintaining parity between the build of `scale` and `scale-logstash` images. I'd recommend we accomplish this by implementing the following behavior:

* Update `bootstrap.py` to assume a default `LOGSTASH_DOCKER_IMAGE` unless set to a non-empty string. The default should be based off of `MARATHON_APP_DOCKER_IMAGE` with `repo/scale:tag` updated to be `repo/scale-logstash:tag`